cmd/commands/runtime.go

```diff
@@ -434,7 +437,11 @@ func ensureIngressHost(cmd *cobra.Command, opts *RuntimeInstallOptions) error {
 		}
 	}
 
-	parsed, err := url.Parse(opts.IngressHost)
+	host := opts.IngressHost
+	if opts.InternalIngressHost != "" {
+		host = opts.InternalIngressHost
```

Comment:
I don't think it's a good idea to use internal ingress as the external ingress if external is not provided.